<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kamakura</class>
 <widget class="QMainWindow" name="kamakura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>677</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Kamakura - -</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="CodeEditor" name="plainTextEdit">
        <property name="font">
         <font>
          <family>TeX Gyre Adventor</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
    <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSave_2"/>
    <addaction name="actionSave"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionZoom"/>
    <addaction name="actionZoom_2"/>
    <addaction name="actionSearch_and_Replace"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHowTo"/>
    <addaction name="actionKamakura"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave_2"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionZoom"/>
   <addaction name="actionZoom_2"/>
   <addaction name="separator"/>
   <addaction name="actionSearch_and_Replace"/>
   <addaction name="separator"/>
   <addaction name="actionHowTo"/>
   <addaction name="actionKamakura"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/open.png</normaloff>:/new/prefix1/resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/exit.png</normaloff>:/new/prefix1/resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/saveas.png</normaloff>:/new/prefix1/resources/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionKamakura">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/about.png</normaloff>:/new/prefix1/resources/about.png</iconset>
   </property>
   <property name="text">
    <string>Kamakura</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/copy.png</normaloff>:/new/prefix1/resources/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/paste.png</normaloff>:/new/prefix1/resources/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/cut.png</normaloff>:/new/prefix1/resources/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/zoomin.png</normaloff>:/new/prefix1/resources/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom+</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/zoomout.png</normaloff>:/new/prefix1/resources/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom-</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/save.png</normaloff>:/new/prefix1/resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSearch_and_Replace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/findreplace.png</normaloff>:/new/prefix1/resources/findreplace.png</iconset>
   </property>
   <property name="text">
    <string>Search and Replace</string>
   </property>
   <property name="font">
    <font>
     <family>TeX Gyre Adventor</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionHowTo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/resources/example.png</normaloff>:/new/prefix1/resources/example.png</iconset>
   </property>
   <property name="text">
    <string>HowTo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QTabWidget</extends>
   <header>codeeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
